dense-numericals-src/basic-math/arg-maximum.lisp

```diff
@@ -208,7 +210,11 @@
                  (setq max x
                        max-id id))
                (incf id))
-             max-id)))
+             max-id))
+         (make-view (array)
```

Comment:
Perhaps `(make-view ...)` can be renamed to `(as-displaced-array ...)`?

```lisp
(as-displaced-array (array-like)
  (let ((array (nu:asarray array-like)))
    (make-array (array-dimensions array)
                :element-type (array-element-type array)
                :displaced-to array))))
```

Reply:
Sure! WIll change this in the next version.